hibernate-search-elasticsearch/README.md

```diff
@@ -80,7 +83,10 @@ When you're done playing with "dev-mode" you can run it as a standard Java appli
 
 First compile it:
 
-> mvn package
+> ./mvnw package
+
+Note that this command will start a PostgreSQL instance and an Elasticsearch cluster to execute the tests.
```

Comment:
I don't know if you changed something, but from what I understand:

* PostgreSQL and Elasticsearch are started in the "test-compile" and "process-test-classes" phases
* then LibraryResourceTest, which is an integration test masquerading as a unit test, is run by surefire if not in native mode
* then LibraryResourceInGraalIT, which is an integration test, is run by failsafe if in native mode

I find this very strange and I would expect the following:

* PostgreSQL and Elasticsearch are started in the "pre-integration-test" phase
* then LibraryResourceTest, rename to LibraryResourceIT, is run by failsafe if not in native mode
* then LibraryResourceInGraalIT, which is an integration test, is run by failsafe if in native mode

If you did that, then `./mvnw package` would not run the integration tests anymore. You could use `./mvnw verify` if you need that.

Reply:
That's how it's done everywhere else so I won't change that.